rules/windows/execution_posh_psreflect.toml

```diff
@@ -0,0 +1,118 @@
+[metadata]
+creation_date = "2021/10/15"
+maturity = "production"
+updated_date = "2021/10/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the use of PSReflect in PowerShell scripts. Attackers leverage PSReflect as a library that enables
+PowerShell to access win32 API functions.,
+"""
+false_positives = ["Legitimate Powershell Scripts that make use of PSReflect to access the win32 API"]
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "PowerShell PSReflect Script"
+note = """## Triage and analysis
+### Investigating PowerShell PSReflect Script
+
+PowerShell is one of the main tools in the belt of system administrators for automation, report routines, and other tasks.
+
+PSReflect is a library that enables PowerShell to access win32 API functions in an uncomplicated way. It also helps to
+create enums and structs easily—all without touching the disk.
+
+Although this is an interesting project for every developer and admin out there, it is mainly used in the red team and
+malware tooling for its capabilities.
+
+Detecting the core implementation of PSReflect means detecting most of the tooling that uses windows API through
+PowerShell, enabling the defender to discover tools being dropped in the environment.
+
+#### Possible investigation steps:
+- Check for additional PowerShell logs that indicate that the script/command was run.
+- Gather the script content that may be split into multiple script blocks, and identify its capabilities.
+- If the action is suspicious for the user, check for any other activities done by the user in the last 48 hours.
+- Look for additional alerts involving the host and the user.
+
+### False Positive Analysis
+- Verify whether the script content is malicious/harmful.
```

Comment:
move to `false_positives` field

Reply:
So I should remove it from here or add content to the false_positives field?

Reply:
Seems that the current false_positives content already covers this